pkg/destinations/redshift/redshift.go

```diff
@@ -0,0 +1,67 @@
+package redshift
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+	"github.com/scratchdata/scratchdata/util"
+)
+
+type RedshiftServer struct {
+	Host     string `mapstructure:"redshift_host"`
+	Port     int    `mapstructure:"redshift_port"`
+	Username string `mapstructure:"redshift_user"`
+	Password string `mapstructure:"redshift_password"`
+	Database string `mapstructure:"redshift_database"`
+	Schema   string `mapstructure:"redshift_schema"`
```

Comment:
`Schema` is only used in `getColumnNames()`. Should it be used for all queries?

Reply:
Then, how are we planning to take schema?

Reply:
Yes

Reply:
@poundifdef isn't queries made by user?
how can we add schema their?

Reply:
Good point.

Reply:
this is resolved or do you have any idea ?